admin/code/CMSBatchActionHandler.php

```diff
@@ -179,7 +179,11 @@ protected function actionByName($name) {
 		// Find the action handler
 		$actions = $this->batchActions();
 		if(!isset($actions[$name]['class'])) {
-			throw new InvalidArgumentException("Invalid batch action: {$name}");
+			$action = $actions[$name];
+
+			throw new InvalidArgumentException(
+				require __DIR__ . '/../templates/Exceptions/InvalidBatchActionException.php'
```

Comment:
I don't think the template folders is the right place to keep this. In 99.99% of other cases, templates mean ".ss template rendered by SSViewer used to create html code". At best, it'll be a little confusing. At worst, it will break the template manifest.

Given this is the first instance of this pattern, I'd probably be more comfortable with something a `admin/code/exceptionTemplates` folder.